pyzebra/comm_export.py

```diff
@@ -0,0 +1,137 @@
+path = "C:\\Users\\Jakub\\Desktop\\"

```

Comment:
These lines should be removed, while `path` and `name` should become a function argument (probably, a single one).